export/src/main/java/io/opencensus/export/metrics/package-info.java

```diff
@@ -28,4 +28,4 @@
  * for more details.
  */
 @io.opencensus.common.ExperimentalApi
-package io.opencensus.stats.metrics;
+package io.opencensus.export.metrics;
```

Comment:
Should this be export.stats.metrics to match with the proto package?

Reply:
Make sense, done.